django/db/migrations/operations/base.py

```diff
@@ -85,6 +98,15 @@ def describe(self):
         """
         return "%s: %s" % (self.__class__.__name__, self._constructor_args)
 
+    def formatted_description(self):
+        """
+        Output the describe output in a more informative way.
+        """
+        description = self.describe()
+        if self.category is None:
+            return description
+        return f"{self.category.value} {description}"
```

Comment:
`.value` is unnecessary
```suggestion
        return f"{self.category} {description}"
```